sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -15,93 +15,131 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.apache.http.NameValuePair;
-import org.apache.http.client.ClientProtocolException;
 import org.apache.http.client.HttpClient;
 import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.client.methods.HttpUriRequest;
 import org.apache.http.impl.client.DefaultHttpClient;
 import org.apache.http.message.BasicNameValuePair;
 
+import edu.umich.its.lti.utils.PropertiesUtilities;
+
+
 
 public class SectionsUtilityToolServlet extends HttpServlet {
 
+	private static final String POST = "POST";
+	private static final String GET = "GET";
+	private static final String PUT = "PUT";
 	private static final long serialVersionUID = 7284813350014385613L;
 	private static Log M_log = LogFactory.getLog(SectionsUtilityToolServlet.class);
+	private static final String SYSTEM_PROPERTY_FILE_PATH = "sectionsToolPropsPath";
+	private static final String PROPERTY_CANVAS_ADMIN = "canvas.admin.token";
+	private static final String PROPERTY_CANVAS_URL = "canvas.url";
+	protected Properties canvasProperties = null;
+	private String canvasToken;
+	private String canvasURL;
+	
+	
+	public void init() throws ServletException {
+		M_log.debug("init(): Called");
+		getCanvasCredentials();
+		
+	}
 	
 	protected void doGet(HttpServletRequest request,HttpServletResponse response) throws IOException {
 		M_log.debug("doGet: Called");
-		HttpClient client = new DefaultHttpClient();
-		String targetURI= "https://umich.test.instructure.com";
+		canvasRestApiCall(request, response);
+	}
+	
+	protected void doPost(HttpServletRequest request,HttpServletResponse response) throws IOException {
+		M_log.debug("doPOST: Called");
+		canvasRestApiCall(request, response);
+		
+	}
+	protected void doPut(HttpServletRequest request,HttpServletResponse response) throws IOException {
+		M_log.debug("doPut: Called");
+		canvasRestApiCall(request, response);
+		
+	}
+	
+	
+
+	private void canvasRestApiCall(HttpServletRequest request,
```

Comment:
A method comment would be helpful here -- this method actually handles _any_ get, post, or put request (which isn't clear from the method name)